internal/datanode/flow_graph_manager.go

```diff
@@ -215,6 +221,11 @@ func (fm *flowgraphManager) exist(vchan string) bool {
 	return exist
 }
 
+func (fm *flowgraphManager) existWithOpID(vchan string, opID UniqueID) bool {
```

Comment:
why we can't use chan as opID?
is there any special reason an opID is necessary?